src/components/Sections/SectionChart/SectionLineChart.js

```diff
@@ -129,7 +129,7 @@ const SectionLineChart = ({ data, style, dimensions, legend, chartProperties = {
   const frames = timeFrame !== SUPPORTED_TIME_FRAMES.none ?
     Math.ceil(finalToDate.diff(from, timeFrame, true)) : preparedData.length - 1;
   const currentDate = moment(from);
-  for (let i = 0; i <= frames; i++) {
+  for (let i = 0; i < frames; i++) {
     const formattedDate = timeFrame !== SUPPORTED_TIME_FRAMES.none ?
```

Comment:
looks like -1 should be dropped from `preparedData.length` in the frames preparation